L1Trigger/L1THGCal/plugins/backend/HGCalBackendLayer2Processor3DClustering_SA.cc

```diff
@@ -57,15 +65,21 @@ class HGCalBackendLayer2Processor3DClusteringSA : public HGCalBackendLayer2Proce
     std::pair<const std::vector<edm::Ptr<l1t::HGCalCluster>>&, const std::vector<std::pair<GlobalPoint, double>>&>
         inputClustersAndSeeds{clustersPtrs, seedPositionsEnergy};
     // Outputs
+    l1t::HGCalMulticlusterBxCollection collCluster3D;
+
     std::pair<l1t::HGCalMulticlusterBxCollection&, l1t::HGCalClusterBxCollection&>
         outputMulticlustersAndRejectedClusters{collCluster3D, rejectedClusters};
+    l1t::HGCalMulticlusterBxCollection outputSortedAndTruncatedClusters(collCluster3D_sorted);
```

Comment:
If the output collection is copied into a new local collection and the local collection is filled afterwards instead of the original output, the original output will stay empty.

Reply:
Thanks, hopefully now fixed